components/netty-http-echo-service/src/main/java/org/wso2/performance/common/netty/echo/EchoHttpServer.java

```diff
@@ -133,4 +152,34 @@ public void initChannel(SocketChannel ch) throws Exception {
             workerGroup.shutdownGracefully();
         }
     }
+
+    private KeyManagerFactory createSSLContextFromKeystores(File keyStoreFile) {
```

Comment:
This returns `KeyManagerFactory`, the method name mentions about `SSLContext`. Please provide a better name.